cmd/istioctl/main.go

```diff
@@ -38,20 +56,33 @@ var (
 	schema model.ProtoSchema
 
 	postCmd = &cobra.Command{
-		Use:   "create [type] [name]",
-		Short: "Create a configuration object",
+		Use:   "create",
+		Short: "Create configuration objects",
 		RunE: func(c *cobra.Command, args []string) error {
-			if len(args) != 2 {
-				return fmt.Errorf("provide configuration type and name")
+			if len(args) != 0 {
+				return fmt.Errorf("create takes no arguments")
 			}
-			if err := setup(args[0], args[1]); err != nil {
-				return err
-			}
-			v, err := readInput()
+			// Initialize schema global
+			varr, err := readInputs()
 			if err != nil {
 				return err
 			}
-			return cmd.Client.Post(key, v)
+			if len(varr) == 0 {
+				return errors.New("Nothing to create")
+			}
+			for _, v := range varr {
+				fmt.Printf("About to setup using %v with name %v\n", v.Type, v.Name)
+				setup(v.Type, v.Name)
+				fmt.Printf("About to Post rule %v\n", v.Rule)
+				err = cmd.Client.Post(key, v.Rule)
+				if err != nil {
+					return err
+				}
+				fmt.Printf("Posted rule\n")
```

Comment:
I suggest removing the prior two fmt (they are for debugging) and use this one. Specifically we could say "created %v '%v'", type, name